src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -251,6 +317,21 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import com.neurologic.http.impl.ApacheHttpClient; // OK
+ * import static java.awt.Button.A;
+ * import javax.swing.JComponent; // violation, wrong order caused by above static import
+ *                      // all static imports comes at bottom
```

Comment:
please align it on same indentation as comment above

Reply:
done